db/seeds/development.rb

```diff
@@ -86,7 +86,8 @@
   SignIn::ServiceAccountConfig.find_or_initialize_by(service_account_id: vaid_service_account_id)
 identity_dashboard_service_account_config.update!(service_account_id: vaid_service_account_id,
                                                   description: 'VA Identity Dashboard API',
-                                                  scopes: ['http://localhost:3000/sign_in/client_configs'],
+                                                  scopes: ['http://localhost:3000/sign_in/client_configs',
+                                                           'http://localhost:3000/account-controls/csp-lock'],
```

Comment:
Let's add this seed in your PR that introduces the actual functionality (I know danger will complain but I think it makes more sense there)